libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -600,13 +600,101 @@ namespace SohImGui {
         ImGui::Text("%s", text);
     }
 
-    void EnhancementCheckbox(const char* text, const char* cvarName)
+    void RenderCross(ImDrawList* draw_list, ImVec2 pos, ImU32 col, float sz)
     {
+        float thickness = ImMax(sz / 5.0f, 1.0f);
+        sz -= thickness * 0.5f;
+        pos += ImVec2(thickness * 0.25f, thickness * 0.25f);
+
+        draw_list->PathLineTo(ImVec2(pos.x, pos.y));
+        draw_list->PathLineTo(ImVec2(pos.x + sz, pos.y + sz));
+        draw_list->PathStroke(col, 0, thickness);
+
+        draw_list->PathLineTo(ImVec2(pos.x + sz, pos.y));
+        draw_list->PathLineTo(ImVec2(pos.x, pos.y + sz));
+        draw_list->PathStroke(col, 0, thickness);
+    }
+
+    bool CustomCheckbox(const char* label, bool* v, bool disabled, ImGuiCheckboxGraphics disabledGraphic) {
```

Comment:
i figure we can probably come up with a better prefix than `Custom` at some point but i'm not too worried about it

Reply:
I agree, I wasn't really sure what to name it either. That said, I'd love to clean up the functions in this file a bit more in general and move them into a different file, possibly called `CustomWidgets.cpp` or something similar. I figured it might make more sense to do that after the LUS cleanup though.